src/lib.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(rustc_private, link_args)]
```

Comment:
I suggest copy-and-pasting the UPL license from one of our other projects. That said, you don't *need* to put a copyright license on things: you (well, the College in this case...) automatically owns the copyright. Software licenses are useful because they allow you to voluntarily give up some of the rights that copyright law gives you automatically.

Reply:
No issue adding the license in. Does this need to be in every file header? Or just the `lib.rs`?

Reply:
It needs to be in every file with meaningful content. [As a rough proxy, every `.rs` file.]